app/components/agenda/agendaitem/agendaitem-decision.js

```diff
@@ -405,6 +405,18 @@ export default class AgendaitemDecisionComponent extends Component {
     beslissingPiecePart,
     betreftPiecePart
   ) {
+    if (!report.id) {
```

Comment:
Probably better to check for existence of decisionResultCode instead. Otherwise decisions might get overwritten when saving a report